include/exadg/structure/driver.cpp

```diff
@@ -111,7 +111,10 @@ Driver<dim, Number>::setup()
       AssertThrow(false, dealii::ExcMessage("Not implemented."));
     }
 
-    pde_operator->setup_solver();
+    if(application->get_parameters().problem_type == ProblemType::Unsteady)
+      pde_operator->setup_solver(time_integrator->get_scaling_factor_mass());
```

Comment:
here i needed to get the scaling factor from thetime integrator. hence I made it public, see `time_int_gen_alpha_base.h`